### PR TITLE
feat(webconference): #18 add git action workflow and unit test

### PR DESCRIPTION
**Pull request by AyaNADINI**
## Describe your changes
Adding a gitHub actions that allows you to automate your build and tests for every pull request and commit to your repository.
Adding front and back unit tests that make gradle/node workflow integration work.
Adding dependency for powerMock(2.0.2), mockito(2.+), junitVersion(4.10) tests

## Issue ticket number and link
18 : https://github.com/OPEN-ENT-NG/webconference/issues/18
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequences regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequence feature, I have added thorough tests.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/ts/services/_tests_/SimpleCI.test.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+            it("throwns an exception", () => { });
+        });
+        describe("when a == 0", () => {
+            it("returns 0", () => { });
+        });
+        describe("when a > 0", () => {
+            it("returns xyz", () => { });
+        });
+    });
+});
```
Comment (AyaNADINI) [inline src/main/resources/public/ts/services/_tests_/SimpleCI.test.ts:13]: ceci est un test basique pour prouver que l'intégration workflow node fonctionne

File: .gitignore [hunk 2]
```diff
@@ -8,6 +8,7 @@
 .jshintrc
 node_modules/
 src/main/resources/view
+/coverage/front/cobertura-coverage.xml
```
Comment (flmartineau) [inline .gitignore:11]: Tu peux enlever cette ligne, fait juste attention de pas push le XML

Comment (AyaNADINI) [reply .gitignore:11]: D'accord je l'enlèverai

File: build.gradle [hunk 3]
```diff
@@ -53,10 +56,16 @@ compileTestJava {
     targetCompatibility = project.targetCompatibility
 }
 dependencies {
+    implementation 'org.testng:testng:7.1.0'
+    implementation 'junit:junit:4.13.1'
     compileOnly "io.vertx:vertx-core:$vertxVersion"
-    compile "org.entcore:common:$entCoreVersion"
```
Comment (flmartineau) [inline build.gradle:57]: Cette ligne était inutile ?

**Review by alicedraillard — APPROVED** [review]

**Pull request merged**